amgut/db/patches/0038.sql

```diff
@@ -0,0 +1,362 @@
+-- Jun 22, 2016
+-- Add IMPACT OF PERSONAL MICROBIOME INFORMATION survey
+----------------------------------------------------------
+-- survey_group
+----------------------------------------------------------
+INSERT INTO survey_group (group_order, american, british) VALUES (-5, 'Personal_Microbiome', 'Personal_Microbiome');
+
+----------------------------------------------------------
+-- surveys
+----------------------------------------------------------
+INSERT INTO surveys (survey_id, survey_group) VALUES (5, -5);
+
+----------------------------------------------------------
+-- survey_question
+----------------------------------------------------------
+-- Change so can have the same wording on supplemental questionares, but the short name must still be unique
+ALTER TABLE ag.survey_question DROP CONSTRAINT idx_survey_question;
+ALTER TABLE ag.survey_question DROP CONSTRAINT idx_survey_question_0;
+INSERT INTO survey_question (survey_question_id, question_shortname, american, british) VALUES
+(186, 'PM_GENDER', 'What is your gender?', 'What is your gender?'),
+(187, 'PM_AGE', 'What is your age?', 'What is your age?'),
+(188, 'PM_ETHNICITY', 'What is your ethnicity?', 'What is your ethnicity?'),
+(189, 'PM_EDUCATION', 'What is your highest level of education?', 'What is your highest level of education?'),
+(190, 'PM_HEALTH', 'How good is your health?', 'How good is your health?'),
+(191, 'PM_PARTICIPATION_REASON', 'What prompted you to participate in the American Gut Project? Please select all that apply.', 'What prompted you to participate in the American Gut Project? Please select all that apply.'),
+(192, 'PM_GASTRO_PROBLEMS', 'What type of problems do you have? Please select all that apply.', 'What type of problems do you have? Please select all that apply.'),
```

Comment:
The question in the document Cinnamon sent reads "If you checked 'I have gastrointestinal problems,', ...." Any reason why it's not written like that here? Is it because the question only appears if they respond yes to the previous question?